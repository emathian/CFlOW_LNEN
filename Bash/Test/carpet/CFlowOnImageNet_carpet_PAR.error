Loading pytorch-gpu/py3/1.9.0
  Loading requirement: cuda/11.2 nccl/2.9.6-1-cuda cudnn/8.1.1.33-cuda gcc/8.4.1
    openmpi/4.1.1-cuda intel-mkl/2020.4 magma/2.5.4-cuda sox/14.4.2
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 1024 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 2048 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torchvision-0.10.0a0+300a8a4-py3.9-linux-x86_64.egg/torchvision/transforms/transforms.py:280: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  ../c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
srun: Job 1942318 step creation temporarily disabled, retrying (Requested nodes are busy)
srun: Step created for job 1942318
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 1024 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 2048 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torchvision-0.10.0a0+300a8a4-py3.9-linux-x86_64.egg/torchvision/transforms/transforms.py:280: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
Traceback (most recent call last):
  File "/linkhome/rech/genkmw01/ueu39kt/cflow-ad/main_cflow.py", line 94, in <module>
    main(c)
  File "/linkhome/rech/genkmw01/ueu39kt/cflow-ad/main_cflow.py", line 87, in main
    train(c)
  File "/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/train.py", line 535, in train
    load_weights(encoder, decoders, c.checkpoint)
  File "/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/custom_models/utils.py", line 72, in load_weights
    state = torch.load(path, map_location='cuda:0')
  File "/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/gpfsscratch/rech/ohv/ueu39kt/CFLOW/weights/carpet_parallel_2809_4/carpet/24/mvtec_wide_resnet50_2_freia-cflow_pl3_cb8_inp512_run0_carpet_2022-09-30-14:18:56.pt'
srun: error: jean-zay-ia825: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=1942318.1
