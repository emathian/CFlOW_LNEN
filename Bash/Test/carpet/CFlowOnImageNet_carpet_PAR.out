========================================================================
First test parallel from train parallel file epoch 24
========================================================================
LR schedule: [12, 18, 22]
Number of pool layers = 3
CNF coder: 512
CNF coder: 1024
CNF coder: 2048
img_types   1
img_types   6
train/test loader length 280 117
train/test loader batches 8 4
Loading weights from /gpfsscratch/rech/ohv/ueu39kt/CFLOW/weights/carpet_parallel_2809_4/carpet/24/mvtec_wide_resnet50_2_freia-cflow_pl3_cb8_inp512_run0_carpet_24_0.pt
Len test_loader   4

Compute loss and scores on test set:
step  % :  0.0  i/I =  0 / 4
Epoch: 0 	 step: 0.0000 
Epoch: 0 	 test_loss: 1.2700 and 5.62 fps
DET_AUROC: 	 last: 90.13 	 max: 90.13 	 epoch_max: 0
DET_AUROC  0.9012841091492777
 SEG_AUROC   0.9184058115485275
SEG_AUROC: 	 last: 91.84 	 max: 91.84 	 epoch_max: 0
========================================================================
First test parallel from best checkpoint
========================================================================
LR schedule: [12, 18, 22]
Number of pool layers = 3
CNF coder: 512
CNF coder: 1024
CNF coder: 2048
img_types   1
img_types   6
train/test loader length 280 117
train/test loader batches 8 4
